development/attributions-check.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+yarn generate:attributions
+
+ATTRIBUTION_TXT="./attribution.txt"
```

Comment:
Nit: Should this file be called `attributions.txt`?

Reply:
@mcmire https://github.com/MetaMask/oss-attribution-generator/blob/main/index.js#L246 by the default sets the file name as attribution.txt we can either update the package to take a file name arg. or we can manually rename the file with shell or we just keep it attribution.txt the same way mobile currently works

Reply:
Ah, I see. Never mind then. We can keep the same filename.